api4/post.go

```diff
@@ -396,6 +397,54 @@ func getPost(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getPostsByIds(c *Context, w http.ResponseWriter, r *http.Request) {
+	var postIDs []string
+
+	json.NewDecoder(r.Body).Decode(&postIDs)
+	if postIDs == nil {
```

Comment:
It would probably be a good idea to enforce a limit, the users one doesn't but I think it would be nice. I can probably set it to 60 which is generally the post fetch limit anyway.

Reply:
I'm not sure of the history, but I do see [a limit of 1000 posts for getting posts by channel](https://github.com/mattermost/mattermost-server/blob/master/store/sqlstore/post_store.go#L990). I think a limit like that is totally more than reasonable.